css-typed-om/Overview.bs

```diff
@@ -784,10 +782,20 @@ The following are the arithmetic operations you can perform on dimensions:
         is set to |this|.
 </div>
 
-<div class=issue>
-    Define {{CSSNumericValue/equals()}}.
+<div algorithm="CSSNumericValue.equals(...|value|)">
+    The <dfn for=CSSNumericValue method>equals(...|value|)</dfn> method,
```

Comment:
Note that .equals() is variadic; you have to check that *all* the values are equal to each other/this.